.github/workflows/ci.yml

```diff
@@ -0,0 +1,26 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - develop
+      - main
+  workflow_dispatch:
+
+jobs:
+  job-install:
+    name: Install dependencies
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout current commit (${{ github.sha }})
+        uses: actions/checkout@v3
+
+      - name: Install Dependencies
```

Comment:
@elbica 
근데 `.yarn/cache` 에서 의존성 버전관리 하고 있는거면 의존성 설치 step 없어도 되는거 아니야? git으로 관리 안되는 의존성이 있어서 의존성 다시 설치하는거야?

Reply:
나도 이부분이 헷갈려서 로컬 새로 파서 테스트 해봤어

<img width="191" alt="image" src="https://user-images.githubusercontent.com/74011724/204959858-f39ec9e0-0089-406b-8021-11b4cda8a1d0.png">

- 여기 있는 unplugged 폴더 (pnp 모드를 지원하지 않는 라이브러리로 추정)은 yarn install을 해야 생겨
- lint나 dev 실행할 때 이 폴더가 없으면 에러가 나더라구!

yarn 이 권장하는 gitignore : https://next.yarnpkg.com/getting-started/qa#which-files-should-be-gitignored

---
이것저것 시도해보다가.. unplugged 폴더도 깃에 올리고 ci는 install 안하는걸로 수정했어~

Reply:
디코에 올린거 여기에두 올릴게:)

.yarn/unplugged 가 뺄지 말지 문서에서 애매하게 말해서 개인적으로 공부한거 남길게~

> .yarn/unplugged should likely always be ignored since they typically hold machine-specific build artifacts. Ignoring it might however prevent Zero-Installs from working (to prevent this, set enableScripts to false).

`.yarn/unplugged` 에 대한 설명인데, 이 폴더를 무시하면 `zero install`이 안될 수 있다고 나와있긴해
https://next.yarnpkg.com/getting-started/qa#which-files-should-be-gitignored

`.yarn/unplugged` 에 대해 찾아보니까 
yarn pnp에서 패키지를 `zip` 아카이브들로 관리하는데 런타임이 FS(file system)에 올려서(mount) 접근하나봐. mount된 것들은 `read-only`이어서 외부에서 다시 쓰여지거나 하진 않는다고 하더라구.(소올직히 무슨 말인지 모름ㅋㅋㅋ 런타임에 다시 쓰여지지 않는다고 받아들여짐)

근데 몇몇 경우에 이런 패키지를 `read-only`로 유지하는게 어려운데 예를 들어 `postinstall` 스크립트를 실행할 경우라고 해. 이런 경우에 yarn은 특정 패키지를 unpack해서 그들 개별 폴더에 저장하는데 이 특정 패키지를 `unplugged` 하나봐!! 여기에도 설명이 나와 함 읽어봥 
**unplugged package : **https://yarnpkg.com/advanced/lexicon#unplugged-package

`.yarn/unplugged` 에 있는 녀석들은 zip으로 관리되지 않는다고 해~~

이건 `postinstall`에 대한 문서 설명이야
요약하자면 `postinstall` script 가 있는 패키지의 디렉터리 구조가 약간 수정된다고 함! zip으로 관리되던 패키지를 풀어서 실제 디렉터리 상에 둬서` node_modules` 마냥 약간은 불안정하다고 이해가 됩니다!
**about postinstall: ** https://yarnpkg.com/advanced/lifecycle-scripts#a-note-about-postinstall

그래서 일단은 approve하고 나중에 패키지 설치에서 문제 생길 때(의존성이 막 바뀐다든지 등..) 다시 이야기 하자구~

근데 start-storybook 하면 왜 `node_modules`가 생기는거지 pnp가 잘 적용이 안된건가